policies/P.C.09.rego

```diff
@@ -0,0 +1,10 @@
+package main
+
+import data.lib.kubernetes
+
+# https://kubesec.io/basics/spec-hostaliases/
+violation[msg] {
+	kubernetes.pods[pod]
+	pod.spec.hostAliases
+	msg = kubernetes.format(sprintf("The %s %s is managing host aliases", [kubernetes.kind, kubernetes.name]))
```

Comment:
by looking at this message, not very sure what need to do, is this trying to say the pod shouldn't have a hostAliases specified?

Reply:
This is a good point, I think at some point I need to standardize the error messaging. I'll raise a ticket for that.